### DISCUSSION TITLE
Installation of wiki.js via installation script failed

### DISCUSSION THREAD

**Discussion by a101y** [requarks/wiki#5491] [Error / Bug Report] [answered]
Hello everyone,

I am in the process of setting up wiki.js on Ubuntu Server 20.04. Now unfortunately I can't get any further with the installation of wiki.js via the installation script.

The command in the terminal/shell looked like this:
`curl -sSo- https://wiki.js.org/install.sh | bash`

I tried the link and found out that the page is not found.

Is there a new link to the install script and if not are there other solutions?

With kind regards
a101y

**Comment by RVRX** [comment]
@a101y can you link to the install guide you were following?

**Comment by NGPixel** [comment, marked as answer]
That would be for the very old 1.x version. Please follow the instructions for 2.x instead:

https://docs.requarks.io/install
